projects/plugins/jetpack/modules/videopress/class.videopress-options.php

```diff
@@ -5,55 +5,40 @@
  * @package automattic/jetpack
  */
 
+use Automattic\Jetpack\VideoPress\Options as Package_Options;
```

Comment:
Is this whole file going to be removed in a future version?

Reply:
Yes. again only keeping it there to be extra safe